README.md

```diff
@@ -53,6 +53,11 @@ Install the binaries and configuration on your host after you've inspected the a
 tar -xzf podman-linux-amd64.tar.gz
 sudo cp -r podman-linux-amd64/usr podman-linux-amd64/etc /
 ```
+
+### Host configuration
+
+`nsenter`needs to be installed seperately. `uidmap` is also needed to run podman as an unprevilied user.
```

Comment:
@tomdavidson thanks for creating this PR!

Please rephrase this as follows and amend your commit:
```
The following binaries should be installed on your host:
* `iptables`
* `nsenter`
* `uidmap` (for rootless mode)
```